src/embedding_scvi/_components.py

```diff
@@ -46,6 +78,8 @@ def __init__(
         n_in: int,
         n_out: int,
         bias: bool = True,
+        cat_dim: int | None = None,
+        conditional: bool = False,
```

Comment:
Needs documentation

Reply:
It currently fails. Can you add something like batch_key (it's the site in mrVI) and ConditionalNorm is only computed on this. It would be 'assay_suspension' for census.